code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -516,7 +516,7 @@
 	attack_verb = "honk"
 	attack_sound = 'sound/items/airhorn2.ogg'
 	prefix = "Bananium"
-	special_names = list("Peel","Cream Pie","Split","Honk")
+	special_names = null

```

Comment:
this is the default value so just delete the line entirely OR do what I put in that last review comment >:)